### PR TITLE
Make Corgi Spawner spawn Ian Instead

### PR DESCRIPTION
**Pull request by Emisse**
we literally havent had ian this whole time we just had some random ass corgi. Changing the corgi spawner to this because its already mapped so it wont be cbt. this also makes it so cargo cant order corgi crate and get the traitor objective.

### CODE REVIEW

File: Resources/Prototypes/Entities/Mobs/NPCs/pets.yml [hunk 1]
```diff
@@ -32,7 +32,7 @@
       dead: corgi_dead
   - type: Butcherable
     spawned:
-    - id: FoodMeatCorgi
+    - id: FoodMeat
       amount: 2
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Mobs/NPCs/pets.yml:36]: other corgis giving corgi meat is fine, doesnt have to just be ian

**Review by mirrorcult — CHANGES_REQUESTED** [review]

**Review by PaulRitter — APPROVED** [review]
lgtm besides mirrors comment

File: Resources/Prototypes/Entities/Mobs/NPCs/pets.yml (hunk 1 above)
Comment (Emisse) [reply Resources/Prototypes/Entities/Mobs/NPCs/pets.yml:36]: the thing is theyre dropping the traitor objective currently, so cargo just orders corgis and gets infinite traitor objectives

Comment (Emisse) [reply Resources/Prototypes/Entities/Mobs/NPCs/pets.yml:36]: this is something i view as unhealthy for the game as ive seen it happening very often

**Pull request merged**